cvat-core/src/index.ts

```diff
@@ -65,6 +65,7 @@ export default interface CVATCore {
         setAuthData: any;
         removeAuthData: any;
         installedApps: any;
+        apiSchema: any;
```

Comment:
Will it be possible to add something more specific than `any`?

Reply:
We do not need to define precise type for the whole scheme, but at least what we should do it for things we use on client